sample.html

```
@@ -0,0 +1,19 @@
+!!DOCTYPE html>
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)